### ISSUE TITLE
Edge directions and merging `citing' and `cited-by'

### ISSUE THREAD

**Issue by pesho-ivanov** [CitationGecko/gecko-react#11]
The two views (papers _citing_ and _cited by_ the seeds) can naturally be visualized together. Such a view will not only show more relevant information but will also simplify the user interaction by skipping the choice.

Any (or both) of these two approaches can be used to distinguish the _citing_ from the _cited by_ nodes:
1. Adding direction to the edges (which will also allow for clean semantics for non-seed dependence #17)
2. DAG'ifying the graph (as cycles are possible) and plotting it left-to-right or top-to-bottom.

**Comment by bjw49** [comment]
I think option 2 is the more natural solution. 

Related ticket: #31

**Comment by no-identd** [comment]
I agree with the DAGification as the better option, but I'd also like to add a note:
If you do end up adding directed edges of some form, there's UX research (I can dig up the citations IFF you insist.) showing that tapering edges (thick at the start, thin at the end) function as much better visual indicators of direction than arrows do.
